tests/api/switch_api_test.py

```diff
@@ -320,7 +320,7 @@ def test_anonymous_switch_can_be_netman_proxied(self):
             username='root',
             password='password',
             port=None,
-            netman_server='1.2.3.4').once().ordered().and_return(self.switch_mock)
+            netman_server=['1.2.3.4']).once().ordered().and_return(self.switch_mock)
```

Comment:
I am sorry but I have to agree with @mat128 since netman is now public we can't only take into account our own utilization of the library. We should try our best to have as little impact as possible on the other users. Backward compatibility is easy to achieve in this case so I would got the extra mile.
:-1: